pom.xml

```diff
@@ -99,10 +96,10 @@
 		<reportingUIVersion>1.6.0</reportingUIVersion>
 		<serializationxstreamVersion>0.2.14</serializationxstreamVersion>
 		<uicommonsVersion>2.6.1</uicommonsVersion>
-		<uiframeworkVersion>3.13.0</uiframeworkVersion>
+		<uiframeworkVersion>3.15.0</uiframeworkVersion>
 		<uilibraryVersion>2.0.6</uilibraryVersion>
 		<ugandaemrfingerprintVersion>1.0.6-SNAPSHOT</ugandaemrfingerprintVersion>
```

Comment:
@dbaluku For UgandaEMR fingerprint use 1.0.6 which is the latest released version

Reply:
Done